engine/src/test/java/net/jqwik/engine/properties/shrinking/UnshrinkableTests.java

```diff
@@ -0,0 +1,33 @@
+package net.jqwik.engine.properties.shrinking;
+
+import net.jqwik.api.Arbitraries;
+import net.jqwik.api.Example;
+import net.jqwik.api.Property;
+import net.jqwik.api.arbitraries.SizableArbitrary;
+import org.assertj.core.api.Assertions;
+
+import java.util.Random;
+import java.util.Set;
+
+class UnshrinkableTests {
+	@Example
+	void nullValueEquals() {
+		Unshrinkable<?> unshrinkable1 = new Unshrinkable<>(null);
+		Unshrinkable<?> unshrinkable2 = new Unshrinkable<>(null);
+		Assertions.assertThat(unshrinkable1.equals(unshrinkable2)).isTrue();
+	}
+
+	@Example
+	void nullValueHashCode() {
+		Unshrinkable<?> unshrinkable = new Unshrinkable<>(null);
+		Assertions.assertThat(unshrinkable.hashCode()).isEqualTo(0);
+	}
+
+	@Property(tries = 50)
+	void nullValueUnshrinkable() {
```

Comment:
In properties you should never use `new Random()` directly because this will prevent deterministic repeats. Instead use a parameter `@ForAll Random random` and use that.

Reply:
@jlink
Thank you for your review. I have amended your request.